.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+
+node_js:
+  - "4.1"
+  - "4.0"
+  - "0.12"
+  - "0.11"
+  - "0.10"
```

Comment:
@XVincentX 

We are not there yet, Apiary still runs 0.10; as long as Apiary runs 0.10 we have to fully support the version.